erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md

```diff
@@ -0,0 +1,69 @@
+<!-- add-breadcrumbs -->
+# Opening Balance in Accounts
+
+**The opening balance is the balance that is brought forward at the beginning of an accounting period from the end of a previous accounting period or when starting out.**
+
+## 1. Introduction
+
+If you are a new company you will have minimal opening balances to be imported. However, if you are migrating from a legacy accounting system like Tally or a Fox Pro based software you will have considerable data to be imported as opening balance.
+
+We recommend that you start using ERPNext for accounting from a new financial year, but you could start midway too. To setup your accounts, you will need the following for the “day” you start using accounting in ERPNext:
+
+##### Assets:
```

Comment:
do a pull on frappe_theme and see if the heading levels seem ok, the sizes were changed
adding numbers to sub headings would look nice, 1.1, 1.2....